### ISSUE TITLE
Rules for special characters

### ISSUE THREAD

**Issue by OleSan** [inma-no/advertsspec#30]
We get a few creatives where the there are problems with special characters. Should we recommend that they are written in HTML code?

Example:
```
å = &aring; 
```

That should guarantee that it will work on all sites. Most should run with UTF-8 today, but you're not sure if you make it for more than one site.  

Included meta info in HTML files also get ignored by AdTech, at least when you upload it. It can be manually set by the traffic manager/ad operations manager, but that is an other source of human error when uploading it to the site. 

Will this be to cumbersome for creative agencies? Is there a smarter way to go about this?

**Comment by gregersrygg** [comment]
Yes, it could be a good idea to escape non-7-bit ASCII characters, but not all strings should be html-escaped. It depends on the output, but html is probably what is mostly used by ads.

**Comment by OleSan** [comment]
So if we say something like this?

> In HTML code, non-7-bit ASCII characters should be HTML escaped.
 ```
Ex:
På = P&aring;
```

**Comment by gadgetgnome** [comment]
Maybe also add that one should avoid special chars in code like @ & etc. We got some feedback from Tor Harald. Also that code shouldn't be minified.
